### PR TITLE
Show custom facility fields when in embed mode

### PR DESCRIPTION
**Pull request by caseycesari**
## Overview

When viewing the details of a facility in embed mode, render the custom fields in the details sidebar.

Only the custom fields that were added by the contributor to a facility are displayed. Custom fields added by another user to the same facility will not be displayed.

Connects #1334

## Demo

*The `test` field is not displayed because it has a null value*
image

Screen Shot 2021-05-17 at 4 03 28 PM

## Testing Instructions

- Log in as a contributor, and go to Settings > Embed
- View the example map, and find a facility that you have added custom fields to.
- View the facility details, and ensure the custom fields are shown in the sidebar. 
- Ensure any fields that have a `null` value are not displayed.
- Toggle some of the custom field checkboxes in the "Include these fields" config section, and ensure the fields that are unchecked do not appear in the sidebar.

## Checklist

- [x] `fixup!` commits have been squashed
- [x] CI passes after rebase
- [x] CHANGELOG.md updated with summary of features or fixes, following [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) guidelines

### CODE REVIEW

File: src/app/src/components/FacilityDetailsStaticMap.jsx [hunk 1]
```diff
@@ -36,7 +36,7 @@ function FacilityDetailsStaticMap({
                     lat,
                     lng,
                 })}
-                alt={`Facility ${name} at latitide ${lat} and longitude ${lng}`}
+                alt={`Facility ${name} at latitude ${lat} and longitude ${lng}`}
```
Comment (caseycesari) [inline src/app/src/components/FacilityDetailsStaticMap.jsx:39]: Just a quick typo fix that I noticed.

File: src/app/src/components/FacilityDetailSidebar.jsx [hunk 2]
```diff
@@ -95,7 +96,10 @@ const detailsSidebarStyles = Object.freeze({
 
 class FacilityDetailSidebar extends Component {
     componentDidMount() {
-        return this.props.fetchFacility();
+        return this.props.fetchFacility(
+            Number(this.props.embed),
+            this.props.user.user.id,
```
Comment (caseycesari) [inline src/app/src/components/FacilityDetailSidebar.jsx:101]: I was hoping to bind these in `mapDispatchToProps`, but as far as I could tell, you can't reference props that are bound in `mapStateToProps` in `mapDispatchToProps`.

File: src/app/src/components/FacilityDetailSidebar.jsx [hunk 3]
```diff
@@ -95,7 +96,10 @@ const detailsSidebarStyles = Object.freeze({
 
 class FacilityDetailSidebar extends Component {
     componentDidMount() {
-        return this.props.fetchFacility();
+        return this.props.fetchFacility(
+            Number(this.props.embed),
```
Comment (caseycesari) [inline src/app/src/components/FacilityDetailSidebar.jsx:100]: The API expects `1` or `0`, not `true` or `false`.

**Review by TaiWilkin — APPROVED** [review]
Nice work on this! It's working well.

File: src/app/src/components/FacilityDetailSidebar.jsx [hunk 4]
```diff
@@ -236,6 +240,11 @@ class FacilityDetailSidebar extends Component {
             return null;
         };
 
+        const contributorFields = filter(
```
Comment (TaiWilkin) [inline src/app/src/components/FacilityDetailSidebar.jsx:243]: This is a nice clean filter to handle the null value

**Comment by caseycesari** [comment]
Thanks!

**Pull request merged**